docs/modules.rst

```diff
@@ -0,0 +1,5 @@
+zeppelin
```

Comment:
This file has no content and it's not mentioned in the index toctree. You should either remove it or fill it with content AND add it to the toctree in index.rst